src/Animations/Animations/Behaviors/BeginAnimationBehavior.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace FormsCommunityToolkit.Animations
+{
+    public class BeginAnimationBehavior : Behavior<VisualElement>
+    {
+        private static VisualElement associatedObject;
+
+        protected override async void OnAttachedTo(VisualElement bindable)
+        {
+            base.OnAttachedTo(bindable);
+            associatedObject = bindable;
+
+            if (Animation != null)
```

Comment:
Try to only resort to positive checks... in other words change this to: if(Animation == null) return;
Is one of the code guidelines ;)